### PR TITLE
Handle empty evaluation_level for backwards compatibility

### PR DESCRIPTION
**Pull request by salma-elshafey**
Fix: Handle empty string evaluation_level in TaskCompletionEvaluator

Production failures occurred when evaluation_level is passed as '' (empty string) by the platform -not yet updated with the multi-turn changes- which isn't a valid EvaluationLevel enum value. The fix treats empty strings the same as None, defaulting to auto-detect mode instead of raising an error.

### CODE REVIEW

File: assets/evaluators/builtin/customer_satisfaction/evaluator/_customer_satisfaction.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def _resolve_evaluation_level(
     :rtype: Optional[EvaluationLevel]
     """
     valid = [level.value for level in EvaluationLevel]
-    if evaluation_level is None:
+    if evaluation_level is None or evaluation_level == '':
```
Comment (imatiach-msft) [inline assets/evaluators/builtin/customer_satisfaction/evaluator/_customer_satisfaction.py:99]: nit: maybe add some tests to validate this edge case, can be in separate PR

**Review by imatiach-msft — DISMISSED** [review]

File: assets/evaluators/builtin/customer_satisfaction/evaluator/_customer_satisfaction.py (hunk 1 above)
Comment (salma-elshafey) [reply assets/evaluators/builtin/customer_satisfaction/evaluator/_customer_satisfaction.py:99]: added

**Review by imatiach-msft — DISMISSED** [review]

**Review by m7md7sien — APPROVED** [review]

**Review by imatiach-msft — APPROVED** [review]
